### DISCUSSION TITLE
Enums purpose

### DISCUSSION THREAD

**Discussion by renatoperes01** [laravel/framework#34036] [General]
Hey guys, please let me know what do you think about add this solution native to the framework?

https://medium.com/@renatoperes01/how-to-handle-enums-in-laravel-without-external-packages-1794f3f3294e

If It sounds interesting for you, It will be a pleasure to make pull request. I think this is an awesome solution to handle enums.

**Comment by terremoth** [comment]
@renatoperes01 PHP has a native Class to handle/simulate Enums, take a look:

https://www.php.net/manual/pt_BR/class.splenum.php

You don't need any package, nor Laravel built-in solution if you just want to make a Class an enum.
